### DISCUSSION TITLE
Could not show the Chinese game name?

### DISCUSSION THREAD

**Discussion by chonghai** [eduardofilo/RG350_adam_image#129] [Ideas & Requests]
When the game name is Chinese word, the game select UI can not show the right word.
could you fix it? 
thanks!

**Comment by eduardofilo** [comment]
Do you mean SimpleMenu?

**Comment by chonghai** [comment]
> Do you mean SimpleMenu?

Yes. Does SimpleMenu support unicode(UTF8)? if so , how to?
thanks!

**Comment by eduardofilo** [comment]
That should be answered by the SimpleMenu developer. I think someone asked to support Chinese characters in the past.
